integration-tests/chopsticks/src/command/stateTransition.ts

```diff
@@ -0,0 +1,16 @@
+import { execa } from 'execa'
+
+export async function stateTransition(endpoint: string, blockNumber?: number) {
+	const options = ['chopsticks', 'run-block', `--endpoint=${endpoint}`, '--html', '--open']
+
+	if (blockNumber) {
+		options.push(`--block=${blockNumber}`)
+	}
+
+	try {
+		const { stdout } = await execa('yarn', options)
+		console.log(stdout)
+	} catch (error) {
+		console.error('Error:', error)
+	}
```

Comment:
There's two ways of writing this; which one you use is a matter of preference.

```suggestion
	await execa('yarn', options)
	  .then(({stdout})=>{console.log(stdout)})
	  .catch(error => {console.error('Error:', error)})
```

Reply:
Yeah then()/catch() are methods on promises and allow you to handle their results.